src/main/resources/swagger/listings/workspaces

```diff
@@ -9,7 +9,7 @@
 			"method": "GET",
 			"summary": "Get workspace ACL",
 			"notes": "",
-			"type": "string",
+			"type": "WorkspaceACL",
```

Comment:
Yes, it's just these two methods (get/update ACL) and the new model objects. I tried making an enum for ```WorkspaceAccessLevel ``` so people know what the acceptable values are, but apparently that doesn't work.